content/post/on-crypto.md

```diff
@@ -0,0 +1,84 @@
+toxic than most open source toxicity. Absolute massive dealbreaker. The amount of money
+sloshing around there seems to be attracting the kinda folks I don't really want in
+my communities.
+
+## Environmental impact of proof of work
+
+Difficult for me to get past the environmental impact of most proof of work systems. Any
+renewable energy they use is energy that could be used to take more fossil fuels off
+the grid. I don't want proof of work systems to switch to renewable energy - I want them
+to stop existing.
+
+## The rich get richer
+
+In the economic models of almost all 'crypto', growing inequality is a feature, not a bug.
+This is outright formalized in Proof of Stake systems, and is de-facto true in most
+Proof of Work systems. I don't really want to live in a world that is even more unequal
+than the one I live in now.
+
+This seems to be an active area of research, and I'm excited to see newer work (like
+Filecoin) come out of this, although I've no idea if they can ever solve their latency
+problems. Fixes to this would be fundamental to any possible positive outcome from
+the 'crypto' community.
+
+## "Distributed Ledger" is not a solution to many problems
+
+The buzzwordiness of 'blockchain', 'crypto' and 'web3' mean they're now being
+forcefit into problems they are very poor solutions for.
+Every time I see the word 'blockchain', I mentally replace it with 'distributed ledger',
+and see if it can still be a good fit for the given problem. Most often, it does not.
+Problems often have to be reframed in economic terms to have a blockchain solve them,
+and I think this reframing is often fundamentally problematic. When all you have is
+a distributed ledger, everything looks like a problem of incentives.
+
+So any time something purports to solve a problem that is *not* use as a
+currency using a blockchain, consider carefully the problem it is solving, and
+seeing if the solutions it is proposing actually solve the problem in a way that
+is generally usable given latency and throughput consideration.
+
+## Let's evaluate the tech on a case-by-case basis
+
+The [fediverse](https://fediverse.party/) is quite cool. I think of [IPFS](https://ipfs.io/)
+as the spiritual successor to bittorrent in many ways (yes, I know they consider themselves
+to be `web3`). There's a lot of really cool conceptual work happening in
+removing ourselves from being beholden to a couple of major corporations that doesn't
+necessarily require buying into a speculative, environmentally disastrous property bubble.
+I want to evaluate each tech that might be solving a problem I find important, but if
+it has brought in strongly to the 'crypto' or 'web3' hype, it just has a much bigger
+bar to clear to prove that it just isn't a speculative ponzi scheme.
+
+I can't wait for the huge amounts of money and the hype train to die out, so we can
+pick up the useful tech that comes out of this.
```

Comment:
👍 👍 👍

Reply:
Going to add 'and proof of work', because I *do* want that to die out.